spec/std/crystal/hasher_spec.cr

```diff
@@ -252,16 +252,11 @@ describe "Crystal::Hasher" do
       1_i32.hash.should eq(1_f64.hash)
     end
 
-    pending_win32 "should 1_f32 and 1.to_big_f hashes equal" do
-      1_f32.hash.should eq(1.to_big_f.hash)
-    end
-
```

Comment:
What I had said about hashes applies here more so, even if it doesn't apply above.

Reply:
Not necessarily since these assertions are within the _normalization of numbers_ `describe` block and for such case IMO it make sense to have 'em there.